doc/manuals/orion-api.md

```diff
@@ -1236,23 +1239,15 @@ Content-Type is `application/json`
 
 #### Retrieve Entity Attributes [GET /v2/entities/{entityId}/attrs]
 
-This request is similar to retrieving the whole entity, however this one omits the `id` and `type`
-fields.
-
-Just like the general request of getting an entire entity, this operation must return only one
-entity element. If more than one entity with the same ID is found (e.g. entities with
-same ID but different type), an error message is returned, with the HTTP status code set to
-409 Conflict.
```

Comment:
This paragraph has not been moved to the payload or responde code section, but I think it is already covered in "Error Responses" section by this part:

> Ambiguity due to the request may refer to several resources, e.g. attempt to update an entity providing only its ID and several entities with that ID exist, results in `TooManyResults` (`409`).

Reply:
I would leave the first paragraph, which describes the request in a really concise way

```
This request is similar to retrieving the whole entity, however this one omits the `id` and `type`
fields.
```

Reply:
Probably is a good idea to keep the warning of the second paragraph under the second bullet point in Response code section, since requests using the path `/v2/entities/{entityId}/attrs` are more error prone because the entity type is not provided

Reply:
> I would leave the first paragraph, which describes the request in a really concise way
> 
> ```
> This request is similar to retrieving the whole entity, however this one omits the `id` and `type`
> fields.
> ```

You are right. My removal was too aggressive :) Please recover it

Reply:
> I would leave the first paragraph, which describes the request in a really concise way
> 
> ```
> This request is similar to retrieving the whole entity, however this one omits the `id` and `type`
> fields.
> ```

This part was restored



> Probably is a good idea to keep the warning of the second paragraph under the second bullet point in Response code section, since requests using the path `/v2/entities/{entityId}/attrs` are more error prone because the entity type is not provided

This idea was dropped